app/src/main/java/org/simple/clinic/patient/Patient.kt

```diff
@@ -245,7 +245,9 @@ data class Patient(
             CASE
               WHEN
                 (BP.systolic >= 140 OR BP.diastolic >= 90
-                OR BloodSugar.bloodSugarValue IS NOT NULL
+                OR (BloodSugar.type = "random" AND BloodSugar.value >= 300)
+                OR (BloodSugar.type = "fasting" AND BloodSugar.value >= 200)
```

Comment:
Can you add test for high `Random`?

Reply:
I added `1 Normal Random Blood Sugar`. Should I add a blood sugar which crosses the cut off as well?

Reply:
Added a test for high `Random Blood Sugar`